docs/database/usage.mdx

```diff
@@ -316,12 +315,14 @@ then it is possible to connect to this using the `useDatabaseEmulator` method. E
 Ensure you have enabled network connections to the emulators in your apps following the emulator usage instructions in the general FlutterFire installation notes for each operating system.
 
 ```dart
+import 'dart:io';
+
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp();
 
   // Ideal time to initialize
-  FirebaseDatabase.instance.useDatabaseEmulator('localhost', 9000);
+  FirebaseDatabase.instance.useDatabaseEmulator(Platform.isAndroid ? '10.0.2.2' : 'localhost', 9000);
```

Comment:
We tend to use `defaultTargetPlatform == TargetPlatform.android && !kIsWeb` from Flutter rather than `dart:io`